src/ai/backend/gateway/vfolder.py

```diff
@@ -134,6 +136,16 @@ def vfolder_check_exists(handler):
     params = await request.json()
     log.info(f"VFOLDER.CREATE (u:{access_key})")
     assert _rx_slug.search(params['name']) is not None
+    # Resolve host for the new virtual folder.
+    folder_host = params.get('host', None)
+    if not folder_host:
+        folder_host = \
+            await request.app['config_server'].etcd.get('volumes/_default_host')
+        if not folder_host:
+            raise VFolderCreationFailed
```

Comment:
It would be better to add a short message to the exception describing the situation, e.g., "You must specify the vfolder host because the default vfolder host is not configured."